src/utils/constants.ts

```diff
@@ -30,14 +37,66 @@ export enum OnboardingSubject {
   Office365CalendarConnected = 'office365_calendar_connected',
 }
 
+export enum PaymentNotificationType {
+  SEND_TOKENS = 'send-tokens',
+  RECEIVE_TOKENS = 'receive-tokens',
+}
+
+export enum CurrencySymbol {
```

Comment:
why are defining currency symbols when Intl already gices us this ? Why not use the existing parse currency helper ?